automatic/webstorm/tools/ChocolateyInstall.ps1

```diff
@@ -6,7 +6,9 @@ $toolsDir     = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 $programFiles = (${env:ProgramFiles(x86)}, ${env:ProgramFiles} -ne $null)[0]
 $pp = Get-PackageParameters
 
-$installDir = "$programFiles\JetBrains\WebStorm $env:ChocolateyPackageVersion"
+# If the release version ends in '.0', remove this since JetBrains don't
+# use this part of the version number when creating the installation directory.
+$installDir = "$programFiles\JetBrains\WebStorm $($env:ChocolateyPackageVersion -replace '.0$', '')"
 if ($pp.InstallDir) {
     $installDir = $pp.InstallDir
 }
```

Comment:
Same [comment as over on the datagrip PR](https://github.com/chocolatey-community/chocolatey-packages/pull/2366#discussion_r1434491333):

We're creating the directory further down on line 19. Can we not just remove that line and let the installer install wherever it thinks it should? Why do we need to create the directory?